App/Redux/PreferencesRedux.ts

```diff
@@ -34,11 +37,13 @@ export type PreferencesState = {
   publicKey?: string
   profile?: Profile
   pending?: string
+  uiSettings: {[index: string]: string}
```

Comment:
We loose the benefit of TS by making this object be any `string` -> `string`. We always have to know which settings we support, so might as well be explicit about it and type it, similar to the `verboseUi` setting. Related, if we want to group 'ui' settings, i guess we should migrate `verboseUi` into it.